scripts/configs/behavior_pick_place_data_collection.yaml

```diff
@@ -0,0 +1,45 @@
+# An example configuration file.
```

Comment:
change this top-level comment to instead include an example command for how to call this yaml file.